packages/trpc/server/routers/viewer/bookings/editLocation.handler.ts

```diff
@@ -118,6 +118,12 @@ export const editLocationHandler = async ({ ctx, input }: EditLocationOptions) =
         },
         data: {
           location,
+          metadata: location.startsWith("http")
+            ? {
+                ...(typeof booking.metadata === "object" ? booking.metadata : {}),
+                videoCallUrl: location,
+              }
+            : {},
```

Comment:
This condition will wipe out al previous booking metadata which I don't think is desirable.

Reply:
Add a suggestion https://github.com/calcom/cal.com/pull/16011#issuecomment-2261267087